docs/intro/quick_overview.md

```diff
@@ -1,55 +1,59 @@
 # GPT-Engineer Documentation
-
-GPT-Engineer is a project that uses GPT-4 to automate the process of software engineering. It includes several Python scripts that interact with the GPT-4 model to generate code, clarify requirements, generate specifications, and more.
+GPT-Engineer is a project that uses LLMs (such as GPT-4) to automate the process of software engineering. It includes several Python scripts that interact with the LLM to generate code, clarify requirements, generate specifications, and more.
 
 <br>
 
 ## Core Components
-### 1. AI Class (`gpt_engineer/ai.py`)
-The AI class is the main interface to the GPT-4 model. It provides methods to start a conversation with the model, continue an existing conversation, and format system and user messages.
+
+(ai_class)=
```

Comment:
Is this some markdown extension I'm not aware of? Never seen before 😮

Reply:
Me neither! Apparently, its Sphinx way of doing cross-references in the MyST Markdown dialect ([see docs](https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html#explicit-targets)).